__init__.py

```diff
@@ -13,9 +13,11 @@ def defaults(self):
 
     def Run(self):
         kicad_pcb = pcbnew.GetBoard()
-        with open(kicad_pcb.GetFileName().replace('.kicad_pcb', '.brd'), 'wt') as brd_file:
-            convert(kicad_pcb, brd_file)
+        with open(kicad_pcb.GetFileName().replace('.kicad_pcb', '.brd'), 'w') as brd_file:
+            convert_brd(kicad_pcb, brd_file)
+        with open(kicad_pcb.GetFileName().replace('.kicad_pcb', '.obdata'), 'w') as obdata:
```

Comment:
I'm not sure I like that it now unconditionally writes `.obdata` files...

Reply:
I could nest this into an if statement as well, but should a separate plugin also be created so it can still be optional within the KiCad plugin menu?

Reply:
The KiCad Plugin menu now displays two options, one named "Pcbnew to Boardview" that only generates the `.brd` file, and another named "Pcbnew to OpenBoardData" that only generates the `.obdata` file.

Reply:
This works.